history/taskana-simplehistory-provider/src/test/java/acceptance/query/QueryHistoryAccTest.java

```diff
@@ -25,19 +25,19 @@ public QueryHistoryAccTest() {
   @Test
   public void testListValuesAscendingAndDescending() {
     List<String> defaultList =
-        getHistoryService().createHistoryQuery().listValues(HistoryQueryColumnName.COMMENT, null);
+        getHistoryService().createHistoryQuery().listValues(HistoryQueryColumnName.TASK_ID, null);
```

Comment:
Since this is an acceptance test, I would use a more useful testcase like sorting by 'created'.

Reply:
Done